src/locale/en.json

```diff
@@ -918,6 +918,7 @@
     "SizeItem": "Item size",
     "SortByAmmoType": "Ammo Type",
     "SortByAmount": "Stack Size",
+    "SortByArmorElement": "Armor Element",
```

Comment:
Someone who's played D1 please correct me, but I think D1 did not have armor elements, so this should probably have a `(D2)` suffix for consistency.